### DISCUSSION TITLE
请问可以当透明代理网关使用吗？

### DISCUSSION THREAD

**Discussion by m2acgi** [Mygod/VPNHotspot#244] [Q&A]
请问可以当透明代理网关使用吗？谢谢。

**Comment by Mygod** [comment]
What do you mean? Currently this app only supports Android VPN services.

**Reply by m2acgi** [reply]
就是是否支持把 VPNHotspot 所在的主机（手机）作为透明代理服务器或者叫透明网关使用？也就是将局域网内的 PC 网关修改为 VPNHotspot 所在主机的 ip 后, 然后该 PC 所有流量 (tcp, udp, icmp 等) 就能够通过 VPNHotspot

**Reply by Mygod** [reply]
Not supported currently and no plan to support it.
